src/Diagnostics.ModelsAndUtils/Models/ResponseExtensions/IResiliencyReportData.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Diagnostics.ModelsAndUtils.Models.ResponseExtensions
+{
+    public interface IResiliencyReportData
+    {
+        /// <summary>
+        /// Customer's name used for the report's cover. This will normally be either customer's Company or simply Customer's name obtained from the subscription.
+        /// </summary>
+        string CustomerName { get; set; }
+
+        /// <summary>
+        /// Array containing a list of all the resources to be included in the report along with their 
+        /// </summary>
+        ResiliencyResource[] GetResiliencyResourceList();
+
+        /// <summary>
+        /// Array containing a list of all the resources to be included in the report along with their 
+        /// </summary>
+        void SetResiliencyResourceList(ResiliencyResource[] value);
+
+        //      public JObject JSONObject();        
```

Comment:
nit: do you want to remove this comment?

Reply:
Thank you for this. Fixed it.